cruft/_commands/update.py

```diff
@@ -1,29 +1,14 @@
 import json
-import os
 from pathlib import Path
-from shutil import rmtree
 from subprocess import DEVNULL, PIPE, CalledProcessError, run  # nosec
 from tempfile import TemporaryDirectory
-from typing import Any, Dict, Optional, Set
+from typing import Any, Dict, Optional
 
 import click
 import typer
-from cookiecutter.generate import generate_files
-from git import Repo
 
-from .utils import (
-    example,
-    generate_cookiecutter_context,
-    get_cookiecutter_repo,
-    get_cruft_file,
-    is_project_updated,
-    json_dumps,
-)
-
-try:
-    import toml  # type: ignore
-except ImportError:  # pragma: no cover
-    toml = None  # type: ignore
+from . import _diff, _generate
```

Comment:
Can we move this under different sub packages in utils?

Reply:
Done.